### DISCUSSION TITLE
rustdesk --service runs ps aux and loginctl every 500ms

### DISCUSSION THREAD

**Discussion by otonoton** [rustdesk/rustdesk#8696] [General]
### Bug Description

`rustdesk --service` is calling `ps aux` and `loginctl show-session` system commands every 500ms.

Is there a way to turn this off?

### How to Reproduce

Just start rustdesk service

### Expected Behavior

Not forking and executing new processes so often

### Operating system(s) on local side and remote side

Ubuntu 22.04

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![rustdesk-ps-loginctl](https://github.com/user-attachments/assets/ebe86560-cd19-49ab-8635-1f57b9570325)

**Comment by c33s** [comment]
real discussion takes place here https://github.com/rustdesk/rustdesk/discussions/11959
